src/heap.h

```diff
@@ -114,5 +116,92 @@ void Heap<ValueType>::deleteHeap(Node<ValueType>* node) {
     return;
 }
 
+
+template<class ValueType>
+void Heap<ValueType>::show() {}
+
+
+std::string decimalToBinary(int number) {
+    std::string binaryNumber = "";
+    int rest;
+
+    while (number > 0) {
+        rest = number % 2;
+        number /= 2;
+        binaryNumber = std::to_string(rest) + binaryNumber;
```

Comment:
W sumie tutaj masz taką analogiczną metodę to_string(). Można  by było coś podobnego zrobić, ale to jak już uważasz